New Mod Patches

slylycan opened the pull request:
Vanilla Fishing Expanded: VCEF_Fisherman
Vanilla Ideology Expanded - Memes and Structures: VME_Elder
Vanilla Races Expanded - Highmate: VRE_Flirty
Vanilla Races Expanded - Highmate: VRE_Distressed

slylycan committed:
New Mod Patches

[Mod: Trait]
Vanilla Fishing Expanded: VCEF_Fisherman
Vanilla Ideology Expanded - Memes and Structures: VME_Elder
Vanilla Races Expanded - Highmate: VRE_Flirty
Vanilla Races Expanded - Highmate: VRE_Distressed

slylycan committed:
Merge branch 'emipa606:main' into main

slylycan commented:
Have to close due to an error I missed. Will redo

emipa606 commented:
Looks good, Im away for a week or so more before I can do anything about it however. 
Dont worry about formatting, I resave all xml-files as .net xml-objects in PowerShell and save them back to file so make sure the formatting correct and that there are no xml-errors

slylycan committed:
New Mod Patch

Added a patch for "Alpha Mechs" which adds a trait if "Vanilla Factions Expanded - Pirates" is loaded

Trait: VFEP_WarcasketTrait_Mech

emipa606 commented:
Sorry, missed this but should be added now. Thanks!
